<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="8" passed="7" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-02-18T02:12:16 IST" name="Surefire suite" finished-at="2025-02-18T02:13:57 IST" duration-ms="101109">
    <groups>
      <group name="Positive TestCases">
        <method signature="DB_LoginTest.validUserLogin(java.lang.String,java.lang.String)[pri:0, instance:nnd.TestCases.DB_LoginTest@22555ebf]" name="validUserLogin" class="nnd.TestCases.DB_LoginTest"/>
      </group> <!-- Positive TestCases -->
      <group name="Negative TestCases">
        <method signature="DB_LoginTest.inValidUserLogin(java.lang.String,java.lang.String)[pri:0, instance:nnd.TestCases.DB_LoginTest@22555ebf]" name="inValidUserLogin" class="nnd.TestCases.DB_LoginTest"/>
      </group> <!-- Negative TestCases -->
    </groups>
    <test started-at="2025-02-18T02:12:16 IST" name="Surefire test" finished-at="2025-02-18T02:13:57 IST" duration-ms="101109">
      <class name="nnd.TestCases.DB_HomeTest">
        <test-method is-config="true" signature="setUp(java.lang.reflect.Method,org.testng.ITestContext)[pri:0, instance:nnd.TestCases.DB_HomeTest@2a65fe7c]" started-at="2025-02-18T02:13:01 IST" name="setUp" finished-at="2025-02-18T02:13:05 IST" duration-ms="3499" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void nnd.TestCases.DB_HomeTest.verifyPurchaseItem() throws org.apache.poi.openxml4j.exceptions.InvalidFormatException,java.io.IOException,java.lang.InterruptedException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4ffa7041]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifyPurchaseItem()[pri:0, instance:nnd.TestCases.DB_HomeTest@2a65fe7c]" started-at="2025-02-18T02:13:05 IST" name="verifyPurchaseItem" description="HomeTest_TC01" finished-at="2025-02-18T02:13:12 IST" depends-on-methods="nnd.TestCases.DB_LoginTest.validUserLogin" duration-ms="7426" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyPurchaseItem -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:nnd.TestCases.DB_HomeTest@2a65fe7c]" started-at="2025-02-18T02:13:12 IST" name="tearDown" finished-at="2025-02-18T02:13:12 IST" duration-ms="179" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- nnd.TestCases.DB_HomeTest -->
      <class name="nnd.TestCases.DB_LoginTest">
        <test-method is-config="true" signature="setUp(java.lang.reflect.Method,org.testng.ITestContext)[pri:0, instance:nnd.TestCases.DB_LoginTest@22555ebf]" started-at="2025-02-18T02:12:17 IST" name="setUp" finished-at="2025-02-18T02:12:21 IST" duration-ms="4346" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void nnd.TestCases.DB_LoginTest.inValidUserLogin(java.lang.String,java.lang.String) throws org.apache.poi.openxml4j.exceptions.InvalidFormatException,java.io.IOException,java.lang.InterruptedException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4ffa7041]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="inValidUserLogin(java.lang.String,java.lang.String)[pri:0, instance:nnd.TestCases.DB_LoginTest@22555ebf]" started-at="2025-02-18T02:12:21 IST" name="inValidUserLogin" description="LoginTest_TC01 inValid User Login" data-provider="invalidloginData" finished-at="2025-02-18T02:12:29 IST" duration-ms="8395" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[dhan0001]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[dhan#0001]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- inValidUserLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:nnd.TestCases.DB_LoginTest@22555ebf]" started-at="2025-02-18T02:12:29 IST" name="tearDown" finished-at="2025-02-18T02:12:29 IST" duration-ms="164" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp(java.lang.reflect.Method,org.testng.ITestContext)[pri:0, instance:nnd.TestCases.DB_LoginTest@22555ebf]" started-at="2025-02-18T02:12:29 IST" name="setUp" finished-at="2025-02-18T02:12:34 IST" duration-ms="4319" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void nnd.TestCases.DB_LoginTest.inValidUserLogin(java.lang.String,java.lang.String) throws org.apache.poi.openxml4j.exceptions.InvalidFormatException,java.io.IOException,java.lang.InterruptedException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4ffa7041]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="inValidUserLogin(java.lang.String,java.lang.String)[pri:0, instance:nnd.TestCases.DB_LoginTest@22555ebf]" started-at="2025-02-18T02:12:34 IST" name="inValidUserLogin" description="LoginTest_TC01 inValid User Login" data-provider="invalidloginData" finished-at="2025-02-18T02:12:42 IST" duration-ms="8416" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[dhan0002]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[dhan#0002]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- inValidUserLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:nnd.TestCases.DB_LoginTest@22555ebf]" started-at="2025-02-18T02:12:42 IST" name="tearDown" finished-at="2025-02-18T02:12:42 IST" duration-ms="144" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp(java.lang.reflect.Method,org.testng.ITestContext)[pri:0, instance:nnd.TestCases.DB_LoginTest@22555ebf]" started-at="2025-02-18T02:12:42 IST" name="setUp" finished-at="2025-02-18T02:12:46 IST" duration-ms="3700" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void nnd.TestCases.DB_LoginTest.validUserLogin(java.lang.String,java.lang.String) throws org.apache.poi.openxml4j.exceptions.InvalidFormatException,java.io.IOException,java.lang.InterruptedException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4ffa7041]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="validUserLogin(java.lang.String,java.lang.String)[pri:0, instance:nnd.TestCases.DB_LoginTest@22555ebf]" started-at="2025-02-18T02:12:46 IST" name="validUserLogin" description="LoginTest_TC01 Valid User Login" data-provider="validloginData" finished-at="2025-02-18T02:12:51 IST" duration-ms="5397" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[dhan01]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[dhan@01]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validUserLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:nnd.TestCases.DB_LoginTest@22555ebf]" started-at="2025-02-18T02:12:51 IST" name="tearDown" finished-at="2025-02-18T02:12:52 IST" duration-ms="160" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp(java.lang.reflect.Method,org.testng.ITestContext)[pri:0, instance:nnd.TestCases.DB_LoginTest@22555ebf]" started-at="2025-02-18T02:12:52 IST" name="setUp" finished-at="2025-02-18T02:12:56 IST" duration-ms="4010" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void nnd.TestCases.DB_LoginTest.validUserLogin(java.lang.String,java.lang.String) throws org.apache.poi.openxml4j.exceptions.InvalidFormatException,java.io.IOException,java.lang.InterruptedException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4ffa7041]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="validUserLogin(java.lang.String,java.lang.String)[pri:0, instance:nnd.TestCases.DB_LoginTest@22555ebf]" started-at="2025-02-18T02:12:56 IST" name="validUserLogin" description="LoginTest_TC01 Valid User Login" data-provider="validloginData" finished-at="2025-02-18T02:13:01 IST" duration-ms="5364" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[dhan6000]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[dhan@6000]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validUserLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:nnd.TestCases.DB_LoginTest@22555ebf]" started-at="2025-02-18T02:13:01 IST" name="tearDown" finished-at="2025-02-18T02:13:01 IST" duration-ms="158" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- nnd.TestCases.DB_LoginTest -->
      <class name="nnd.TestCases.DB_RegisterTest">
        <test-method is-config="true" signature="setUp(java.lang.reflect.Method,org.testng.ITestContext)[pri:0, instance:nnd.TestCases.DB_RegisterTest@36ebc363]" started-at="2025-02-18T02:13:12 IST" name="setUp" finished-at="2025-02-18T02:13:16 IST" duration-ms="3729" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void nnd.TestCases.DB_RegisterTest.invalidUserSignUp(java.lang.String,java.lang.String) throws java.lang.InterruptedException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4ffa7041]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="invalidUserSignUp(java.lang.String,java.lang.String)[pri:0, instance:nnd.TestCases.DB_RegisterTest@36ebc363]" started-at="2025-02-18T02:13:16 IST" name="invalidUserSignUp" description="RegisterTest_TC02 invalid user sign Up " data-provider="invalidUserData" finished-at="2025-02-18T02:13:27 IST" duration-ms="10432" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[dhan01]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[dhan@01]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidUserSignUp -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:nnd.TestCases.DB_RegisterTest@36ebc363]" started-at="2025-02-18T02:13:27 IST" name="tearDown" finished-at="2025-02-18T02:13:27 IST" duration-ms="178" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp(java.lang.reflect.Method,org.testng.ITestContext)[pri:0, instance:nnd.TestCases.DB_RegisterTest@36ebc363]" started-at="2025-02-18T02:13:27 IST" name="setUp" finished-at="2025-02-18T02:13:30 IST" duration-ms="3529" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void nnd.TestCases.DB_RegisterTest.invalidUserSignUp(java.lang.String,java.lang.String) throws java.lang.InterruptedException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4ffa7041]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="invalidUserSignUp(java.lang.String,java.lang.String)[pri:0, instance:nnd.TestCases.DB_RegisterTest@36ebc363]" started-at="2025-02-18T02:13:30 IST" name="invalidUserSignUp" description="RegisterTest_TC02 invalid user sign Up " data-provider="invalidUserData" finished-at="2025-02-18T02:13:41 IST" duration-ms="10381" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[dhan6000]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[dhan@6000]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidUserSignUp -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:nnd.TestCases.DB_RegisterTest@36ebc363]" started-at="2025-02-18T02:13:41 IST" name="tearDown" finished-at="2025-02-18T02:13:41 IST" duration-ms="154" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp(java.lang.reflect.Method,org.testng.ITestContext)[pri:0, instance:nnd.TestCases.DB_RegisterTest@36ebc363]" started-at="2025-02-18T02:13:41 IST" name="setUp" finished-at="2025-02-18T02:13:45 IST" duration-ms="3788" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void nnd.TestCases.DB_RegisterTest.validUserSignUp() throws java.lang.InterruptedException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4ffa7041]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="validUserSignUp()[pri:0, instance:nnd.TestCases.DB_RegisterTest@36ebc363]" started-at="2025-02-18T02:13:45 IST" name="validUserSignUp" description="RegisterTest_TC01 valid user sign Up " finished-at="2025-02-18T02:13:55 IST" duration-ms="10845" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Signup failed expected [Sign up successful.] but found [This user already exist.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Signup failed expected [Sign up successful.] but found [This user already exist.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at nnd.Pages.DB_RegisterPage.clickSignUp(DB_RegisterPage.java:67)
at nnd.TestCases.DB_RegisterTest.validUserSignUp(DB_RegisterTest.java:60)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validUserSignUp -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:nnd.TestCases.DB_RegisterTest@36ebc363]" started-at="2025-02-18T02:13:57 IST" name="tearDown" finished-at="2025-02-18T02:13:57 IST" duration-ms="131" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- nnd.TestCases.DB_RegisterTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
